tracer/test/test-applications/debugger/Samples.Probes/SmokeTests/AsyncLineProbeWithFieldsArgsAndLocalsTest.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Samples.Probes.Shared;
+
+namespace Samples.Probes.SmokeTests
+{
+    [LineProbeTestData(39)]
```

Comment:
Maybe we should also put a line probe in `RunAsync`, just so that we also cover an async method that retuns `Task` (as opposed to `Task<T>`?

Reply:
Done in 737cb8695e83517d762f432a20ac36eb57845a39.